cpp/src/indexing/point_quadtree.cu

```diff
@@ -0,0 +1,514 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <vector>
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+#include <cudf/column/column_factories.hpp>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/table/table_view.hpp>
+#include <cudf/table/table.hpp>
+#include <cudf/utilities/legacy/type_dispatcher.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <cuspatial/point_quadtree.hpp>
+#include <utility/helper_thrust.cuh>
+#include <utility/quadtree_thrust.cuh>
+
+
+typedef thrust::tuple<double, double,double,double,double,uint32_t,uint32_t> quad_point_parameters;
```

Comment:
There are many opportunities to pick about `, ` and other formatting decisions. Maybe we should adopt an automatic beautifier for `cuspatial` starting with this PR? @zhangjianting @harrism what do you think?

Reply:
I'm all for standardizing formatting. @codereport has been working on rapidsai/cudf#3259 which we can probably use once that's done. My personal opinion is to nit-pick until we can standardize all of rapids cpp at the same time, but that's just me.

Reply:
what would be the right format to use? Please feel free to revise the code.

Reply:
@thomcom It seems that your request of change is the only one left but I am not able to locate the change you requested. Can you take a look?